OsmAnd/src/net/osmand/plus/auto/FavoriteGroupsScreen.kt

```diff
@@ -44,11 +44,13 @@ class FavoriteGroupsScreen(
                 .setBrowsable(true)
                 .setOnClickListener { onClickFavoriteGroup(null) }
                 .build())
-        var collectionSize = 1
-        for (group in favoriteGroups) {
-            if (collectionSize == contentLimit) {
-                break
-            }
+        if (contentLimit < 2) {
```

Comment:
Is it possible?

Reply:
Technically yes. If ConstraintManager gives such value